test_runner/regress/test_storage_controller.py

```diff
@@ -66,7 +69,22 @@ def test_storage_controller_smoke(
     - Marking a pageserver offline
     """
 
+    compatibility_postgres_distrib_dir_env = os.environ.get("COMPATIBILITY_POSTGRES_DISTRIB_DIR")
+    assert compatibility_postgres_distrib_dir_env is not None, (
+        "COMPATIBILITY_POSTGRES_DISTRIB_DIR is not set. It should be set to a path with postgres binaries"
+        "(ideally generated by the previous version of Neon)"
+    )
+    compatibility_postgres_distrib_dir = Path(compatibility_postgres_distrib_dir_env).resolve()
+    compatibility_neon_bin_env = os.environ.get("COMPATIBILITY_NEON_BIN")
+    assert compatibility_neon_bin_env is not None, (
+        "COMPATIBILITY_NEON_BIN is not set. It should be set to a path with Neon binaries "
+        "(ideally generated by the previous version of Neon)"
+    )
+    compatibility_neon_bin = Path(compatibility_neon_bin_env).resolve()
     neon_env_builder.num_pageservers = 3
+    neon_env_builder.version_combination = combination
+    neon_env_builder.compatibility_neon_bindir = compatibility_neon_bin
+    neon_env_builder.compatibility_pg_distrib_dir = compatibility_postgres_distrib_dir
```

Comment:
Let's try to make it possible to apply parameterization without changing the test itself (otherwise it's going to be a massive copy-paste). 
Afiar, you can access `request.param` in `neon_env_builder` fixture to get current parameter

Reply:
I believe this approach is implemented now